abbyy_to_epub3/create_epub.py

```diff
@@ -1133,21 +1138,25 @@ def craft_epub(self, epub_outfile="out.epub", tmpdir=None):
         epub.write_epub(epub_outfile, self.book, {})
 
         # run validation on epub
-        if self.args and self.args.epubcheck:
-            self.validate_epub(epub_outfile)
+        if self.debug or (self.epubcheck):
+            self.validate_epub(epub_outfile, level=self.epubcheck)
 
-    def validate_epub(self, epub_file):
+    def validate_epub(self, epub_file, level=None):
         self.logger.debug("Running EpubCheck on {}".format(epub_file))
+        LEVELS = ['warning', 'error', 'fatal']
+        level = level.lower() or self.DEFAULT_EPUBCHECK_LEVEL
+        try:
+            desired_levels = LEVELS[LEVELS.index(level):]
```

Comment:
QUESTION: This seems a little "clever". Why not just check to see if `LEVELS` contains the `level`?